### PR TITLE
[17.0][MIG] purchase_sale_link_by_origin: Migration to 17.0

### PR DESCRIPTION
**Pull request by nguyenminhchien**

### CODE REVIEW

**Review by celm1990 — APPROVED** [review]
LGTM

File: purchase_sale_link_by_origin/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+By default since [Odoo
+14.0](https://github.com/odoo/odoo/commit/5a1645a8f8f3560eb778da90b6160b322ce0722e),
+PO and SO are linked by their order lines.
+
+This module also link them by the PO's Origin field, to cover more
+cases. For example: - If a user cancels a PO, by default the link would
+have been broken; now it won't; - Or if a user manually defines or
+updates the Origin field of a PO, it will be taken into account.
```
Comment (mmequignon) [inline purchase_sale_link_by_origin/readme/DESCRIPTION.md]: formatting is broken here

**Review by mmequignon — APPROVED** [review]
cosmetic changes, pre-approving, since it doesn't affect the module itself

File: purchase_sale_link_by_origin/readme/DESCRIPTION.md (hunk 1 above)
Comment (HviorForgeFlow) [reply purchase_sale_link_by_origin/readme/DESCRIPTION.md]: @nguyenminhchien

**Comment by nguyenminhchien** [comment]
> cosmetic changes, pre-approving, since it doesn't affect the module itself

Updated

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**